components/04-pages/artist-collabo-page/_artist-collabo-page.scss

```diff
@@ -0,0 +1,62 @@
+.artist-collabo__subtitle {
+  @extend %heading-font-bold;
+  color: $artist-collabo-subtitle-color;
+}
+
+.header-label {
+  top: $spacer-md;
+  box-shadow: 5px 5px 5px $brand-gray-transparent;
```

Comment:
Ich würde den Schlagschatten entfernen. Unser CD verwendet bisher keine und das würde sehr fremdartig in Kombination mit den anderen Seiten wirken

Reply:
✅